src/components/DataView/DataViewFilters.tsx

```diff
@@ -109,14 +110,14 @@ function DataViewFilters(props: DataViewFiltersProps) {
 		});
 	}
 
-	const onActiveFiltersChange = function(activeFiltersParam) {
+	const onActiveFiltersChange = function(activeFiltersParam: { value: DataViewProps["activeFilters"] }) {

```

Comment:
Everything looks good, the only thing I noticed is that I believe we might want to do something like this:
```suggestion
	const onActiveFiltersChange = function(activeFiltersParam: { value: DataViewProps["activeFilters"], comparison: string }) {
```
Becuase when the `onActiveFiltersChange` function get's propped down to the `onApply`, it gets passed both value and comparison. Right now it might not trigger an error but later when we update the file that uses the `onApply`, it might.